packages/cxl-ui/src/components/cxl-app-layout.js

```diff
@@ -118,5 +122,13 @@ export class CXLAppLayoutElement extends LitElement {
     registerGlobalStyles(cxlAppLayoutGlobalStyles, {
       moduleId: 'cxl-app-layout-global',
     });
+
+    this.addEventListener('wheel', this._onWheel.bind(this));
```

Comment:
Can we set up and tear down listening with `wide` mode? This doesn't need to listen on mobiles.

Reply:
Fixed